### DISCUSSION TITLE
Support for Timezone [Apr 13]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5518] [Discord Threads]
> 🙋‍♂️ **DaveTeamMember**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Currently, it's not supported. We have it under our roadmap, but may not be immediate  
#  
> 🙋‍♂️ **DaveTeamMember**  
#### Alright. Thanks for your response 🙂 Any rough ETA I can look forward to ?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1085191514352779304)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
